_resources/_styles/results.styl

```diff
@@ -0,0 +1,34 @@
+// SHARE THIS MODULE
+
+.InputContainer {
```

Comment:
Name's too vague.  Are you going to wrap all inputs in this? Call it share container or something more specific.